src/main/java/org/folio/rest/impl/util/DataImportKafkaTopic.java

```diff
@@ -0,0 +1,38 @@
+package org.folio.rest.impl.util;
+
+import org.folio.kafka.services.KafkaTopic;
+
+public enum DataImportKafkaTopic implements KafkaTopic {
+  DI_COMPLETED("di-completed"),
+  DI_EDIFACT_RECORD_CREATED("di-edifact-record-created"),
+  DI_ERROR("di-error"),
+  DI_INITIALIZATION_STARTED("di-initialization-started"),
+  DI_INVENTORY_AUTHORITY_CREATED_READY_FOR_POST_PROCESSING("di-inventory-authority-created-ready-for-post-processing"),
+  DI_INVENTORY_INSTANCE_CREATED_READY_FOR_POST_PROCESSING("di-inventory-instance-created-ready-for-post-processing"),
+  DI_LOG_SRS_MARC_AUTHORITY_RECORD_CREATED("di-log-srs-marc-authority-record-created"),
+  DI_LOG_SRS_MARC_BIB_RECORD_CREATED("di-log-srs-marc-bib-record-created"),
+  DI_MARC_BIB_FOR_ORDER_CREATED("di-marc-bib-for-order-created"),
+  DI_MARC_FOR_UPDATE_RECEIVED("di-marc-for-update-received"),
+  DI_PARSED_RECORDS_CHUNK_SAVED("di-parsed-records-chunk-saved"),
+  DI_RAW_RECORDS_CHUNK_PARSED("di-raw-records-chunk-parsed"),
+  DI_RAW_RECORDS_CHUNK_READ("di-raw-records-chunk-read"),
+  DI_SRS_MARC_AUTHORITY_RECORD_CREATED("di-srs-marc-authority-record-created"),
+  DI_SRS_MARC_AUTHORITY_RECORD_NOT_MATCHED("di-srs-marc-authority-record-not-matched"),
+  DI_SRS_MARC_BIB_INSTANCE_HRID_SET("di-srs-marc-bib-instance-hrid-set"),
+  DI_SRS_MARC_BIB_RECORD_CREATED("di-srs-marc-bib-record-created"),
+  DI_SRS_MARC_BIB_RECORD_MODIFIED_READY_FOR_POST_PROCESSING("di-srs-marc-bib-record-modified-ready-for-post-processing");
```

Comment:
I'm not sure mod-data-import module should be responsible for all DI topics, I'd leave here only the topic that this particular module publishes to

Reply:
removed redundant topics, only mod-di related topics left